packages/guides/tests/unit/Intersphinx/InventoryLoaderTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\Intersphinx;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+use function count;
+use function file_get_contents;
+use function json_decode;
+use function PHPUnit\Framework\assertIsString;
+
+use const JSON_THROW_ON_ERROR;
+
+final class InventoryLoaderTest extends TestCase
+{
+    private InventoryLoader $inventoryLoader;
+    /** @var JsonLoader|MockObject */
```

Comment:
```suggestion
    /** @var JsonLoader&MockObject */
```
This will help phpstan to understand it better. Your object is both a `JsonLoader` and `MockObject`